src/DataModel/IData.ts

```diff
@@ -0,0 +1,6 @@
+import {IMetadata} from "./IMetadata";
+
+export interface IData extends Object
```

Comment:
OK for the name. As for the separation - for UI based apps it may be a must - I wouldn't like to see hypermedia controls displayed in a grid of i.e. products